app/services/service_provider_seeder.rb

```diff
@@ -13,14 +13,21 @@ def run
     service_providers.each do |issuer, config|
       next unless write_service_provider?(config)
       ServiceProvider.find_or_create_by!(issuer: issuer) do |sp|
+        cert_pems = Array(config['certs']).map do |cert|
+          cert_path = Rails.root.join('certs', 'sp', "#{cert}.crt")
+          cert_path.read if cert_path.exist?
+        end.compact
+
         sp.update(
           approved: true,
           active: true,
           native: true,
           friendly_name: config['friendly_name'],
+          certs: cert_pems,
```

Comment:
**question:** how did you pick which `update` to put this in? I'm not sure it matters but just curious.

Reply:
the one below is plain values from the YAML, and this one is hardcoded values, so I included the transformed value here too

Reply:
Gotcha, makes sense!